src/number_theory/primorial.lean

```diff
@@ -0,0 +1,198 @@
+/-
+Copyright (c) 2020 Patrick Stevens. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Patrick Stevens
+-/
+import tactic
+import data.nat.parity
+import data.nat.choose
+
+/-!
+# Primorial
+
+This file defines the primorial function, and proves a bound on its size.
+
+## Notations
+
+We use the notation `n#` for the primorial of `n`: that is, the product of the primes less than
```

Comment:
```suggestion
We use the local notation `n#` for the primorial of `n`: that is, the product of the primes less than
```
So that people know that they can't automatically use it outside this file.